### PR TITLE
Conda Build For Windows

### PR DESCRIPTION
**Pull request by OwenArnold**
* Should run the full CI 
* Should build conda package 
* Should package and push development conda windows packages to anaconda cloud.

### CODE REVIEW

**Comment by OwenArnold** [comment]
@dtasev could you re-review this now. I think it's good to go.

File: appveyor.yml [hunk 1]
```diff
@@ -35,8 +35,7 @@ test_script:
 - C:\Python36-x64\python.exe -m pytest ..\python
 
 deploy_script:
-- echo head repo branch %APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH%
-- ps: if ($env:APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH -ne "master") { return 0}
+- ps: if ($env:APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH -ne "master") { Exit-AppVeyorBuild }
```
Comment (dtasev) [inline appveyor.yml]: Why is this needed? The `deploy_script` shouldn't be getting called on non-master branches?

Comment (OwenArnold) [reply appveyor.yml]: Unfortunately it is. Look at the build history without it.
